tests/rkt_run_pod_manifest_test.go

```diff
@@ -37,20 +38,22 @@ const baseAppName = "rkt-inspect"
 
 func importImageAndFetchHash(t *testing.T, ctx *rktRunCtx, img string) string {
 	// Import the test image into store manually.
+	var stdout, stderr bytes.Buffer
 	cmds := strings.Fields(ctx.cmd())
 	fetchCmd := exec.Command(cmds[0], cmds[1:]...)
 	fetchCmd.Args = append(fetchCmd.Args, "--insecure-skip-verify", "fetch", img)
-	output, err := fetchCmd.Output()
-	if err != nil {
-		t.Fatalf("Cannot read the output: %v", err)
+	fetchCmd.Stdout, fetchCmd.Stderr = &stdout, &stderr
+
+	if err := fetchCmd.Run(); err != nil {
+		t.Fatalf("Cannot read the output: %v\nstdout: %v\nstderr: %v", err, stdout.String(), stderr.String())
 	}
 
 	// Read out the image hash.
-	ix := strings.Index(string(output), "sha512-")
+	ix := strings.Index(stdout.String(), "sha512-")
 	if ix < 0 {
-		t.Fatalf("Unexpected result: %v, expecting a sha512 hash", string(output))
+		t.Fatalf("Unexpected result: %v, expecting a sha512 hash", stdout.String())
 	}
-	return strings.TrimSpace(string(output)[ix:])
+	return strings.TrimSpace(stdout.String()[ix:])
```

Comment:
Would it be convenient to use the function `expectRegexWithOutput` from over here https://github.com/coreos/rkt/blob/ffacd4505242d3764988398bd03cf7ed4a543af9/tests/rkt_tests.go#L212? The call to `string.Index()` wouldn't be needed any more by using an appropriate regex. In this case `sha512-[0-9a-f]{64}` should work.

See the lines after https://github.com/coreos/rkt/blob/ffacd4505242d3764988398bd03cf7ed4a543af9/tests/rkt_privatenet_test.go#L45 for an example.